dali/operators/reader/loader/webdataset_loader.cc

```diff
@@ -328,6 +317,12 @@ void WebdatasetLoader::PrepareMetadataImpl() {
             output_indicies_.push_back(output);
             component.outputs.num++;
             was_output_set[output] = true;
+          } else {
+            std::call_once(multiple_files_single_component, [&]() {
+              DALI_WARN(make_string("Multiple components matching the same output for output nr ",
+                                    output, " at line ", sample.line_number, " file ",
+                                    index_paths_[wds_shard_index]));
```

Comment:
I think the file name should be quoted - most, if not all, modern OSes allow spaces in paths.
```suggestion
                                    output, " at line ", sample.line_number, " file \"",
                                    index_paths_[wds_shard_index], "\"."));
```

Reply:
alright